docs/cado/deploy/azure/azure-deploy.md

```diff
@@ -34,55 +34,13 @@ Once you receive them, continue on to the steps below.
     - Storage Queue Data Reader
     :::
 
-4. Create a storage account and container to stage the Cado VHD locally within your Azure tenancy.  This will be from where the platform will be deployed.  You must copy the image to the same region in which you want to deploy Cado.  You can complete these steps using the Azure portal or the Azure CLI
+4. Identify the latest Azure image `azure_image` from the [public update information](https://cado-public.s3.amazonaws.com/cado_updates_json_v2.json) e.g. `/communityGalleries/cadoplatform-1a38e0c7-afa4-4e0d-9c56-433a12cd67b1/images/CadoResponseV2.0/versions/2.209.0`
```

Comment:
/communityGalleries/cadoplatform-1a38e0c7-afa4-4e0d-9c56-433a12cd67b1/images/CadoResponseV2.0/versions/latest

will give them the latest image for free. think might be easier than the manual step

Reply:
which we should probably end up defaulting to in terraform. which would remove the need for this step

Reply:
https://bitbucket.org/cadosecurity/processor/pull-requests/8812